Gemfile

```diff
@@ -1,3 +1,3 @@
-source 'https://rubygems.org' do
```

Comment:
this style of sourcing within a `do/end` is no longer considered a best practice. Instead, one `source` per file is preferred these days.